packages/compass-sidebar/src/components/multiple-connections/active-connections/active-connection-navigation.tsx

```diff
@@ -49,16 +54,17 @@ interface Match {
   isMatch?: boolean;
 }
 
-type Collection = Connection['databases'][number]['collections'][number];
+type Collection =
```

Comment:
Would it be easier to export `SidebarConnectionItem`, `SidebarDatabaseItem`, `SidebarCollection` from the other package so that we can used here?

Reply:
Addressed this in [cd319b7](https://github.com/mongodb-js/compass/pull/5886/commits/cd319b7cff14d5e4ed26ff4f05b7dbfa9054088f)